sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -21,6 +21,7 @@ import {
 } from "@azure/core-http";
 
 import { getDefaultUserAgentValue } from "@azure/core-http";
+import "@azure/core-paging";
```

Comment:
@jeremymeng we are supposed to do this change in storage (feature/storage branch) after this PR gets merged?

Reply:
yeah